packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -199,6 +199,15 @@ class StackParentData extends ContainerBoxParentDataMixin<RenderBox> {
   }
 }
 
+/// Whether overflowing children should be clipped, or their overflows be
+/// visible.
+enum Overflow {
+  /// Visible overflow.
```

Comment:
This needs to be a more detailed comment. Right now it's literally just the name of the enum value and the name of the enum class, anyone could guess that Overflow.visible means "visible overflow". :-)